libtextsecure/outgoing_message.js

```diff
@@ -503,9 +511,9 @@ OutgoingMessage.prototype = {
               destination,
               socketMessage,
               this.timestamp,
-              outgoingObject.ttl
+              ttl
             );
-            if (outgoingObject.isFriendRequest) {
+            if (!this.isGroup && isFriendRequest && !isSessionRequest) {
               const conversation = ConversationController.get(destination);
               if (conversation) {
                 // Redundant for primary device but marks secondary devices as pending
```

Comment:
is the group approval on 2ndary devices not need or just not addressed here?

Reply:
Hmm i didn't get this question

Reply:
`If this.isGroup === true  && isFriendRequest && !isSessionRequest` then do we not need to make the secondary device as pending? Or does group never touch the friend requests system?

Reply:
Closed groups should never change the friend request states.